<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorSetEditingWidget</class>
 <widget class="QWidget" name="ColorSetEditingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Axes background color</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Background color</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ColorChoiseWidget" name="backgroundColorWidget" native="true"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Graf. primitives color</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="ColorChoiseWidget" name="grafPrimitivesColorWidget" native="true"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Font color</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="ColorChoiseWidget" name="fontColorWidget" native="true"/>
     </item>
     <item row="1" column="1">
      <widget class="ColorChoiseWidget" name="axesBackgroundColorWidget" native="true"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Axes background brush style</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="AnyComboBox" name="axesBackgroundBrushStylewidget" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ColorsListChoiceWidget" name="graphsColorListChoiceWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorChoiseWidget</class>
   <extends>QWidget</extends>
   <header location="global">commonWidgets/colorchoisewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorsListChoiceWidget</class>
   <extends>QWidget</extends>
   <header>commonWidgets/colorslistchoicewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AnyComboBox</class>
   <extends>QWidget</extends>
   <header location="global">commonWidgets/anycombobox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
